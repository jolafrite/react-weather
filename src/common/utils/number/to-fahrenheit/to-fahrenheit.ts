export const toFahrenheit = (degree: number) => {
  return degree * 1.8 + 32;
};
