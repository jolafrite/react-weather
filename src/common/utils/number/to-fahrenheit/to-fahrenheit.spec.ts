import { toFahrenheit } from "./to-fahrenheit";

describe('toFahrenheit', () => {

  test('convert from degree celcius to fahrenheit', () => {
    expect(
      toFahrenheit(23)
    ).toMatchSnapshot()
  });

  test('convert a below 0 temperature', () => {
    expect(
      toFahrenheit(-10)
    ).toMatchSnapshot()
  });

});
