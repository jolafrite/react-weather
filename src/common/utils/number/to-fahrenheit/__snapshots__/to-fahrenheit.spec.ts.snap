// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toFahrenheit convert a below 0 temperature 1`] = `14`;

exports[`toFahrenheit convert from degree celcius to fahrenheit 1`] = `73.4`;
